test/e2e/dr/quorum-restore.go

```diff
@@ -0,0 +1,332 @@
+package dr
```

Comment:
Style: filenames of go should always have underscores, not dashes

Reply:
Done